slt/align.md

```diff
@@ -13,7 +13,7 @@ We describe an approach to scalable mechanistic interpretability of neural netwo
 
```

Comment:
Maybe this is a good spot to summarize the key claim(s) of SLT?  

Idk, call it "**Watanabe's hypothesis**: knowledge to be gained corresponds to singularities in level sets of the loss landscape" (but then less cryptic)

Reply:
Yes agreed